### DISCUSSION TITLE
Stripe-SCA compatibility with Pretix

### DISCUSSION THREAD

**Discussion by mklute-kluteq** [pretix/pretix#2276] [Q&A] [answered]
Hi,

we get the following message in Stripe (see Screenshot)
![stripe-sca](https://user-images.githubusercontent.com/70173516/139795955-2ce743f4-64ef-4e1c-b9f0-ad97d8ade55e.PNG)

What can we do? I have already update the pretix-Version to a newer version (see Screenshot again).
![pretix-versions](https://user-images.githubusercontent.com/70173516/139796186-6a5be976-fb48-4285-83d7-b0001beff648.PNG)
Should we update pretix again to fix this problem or what can we do?

Best regards and thanks for help
Max

PS.: The current update was only released recently. So I'm assuming that this doesn't fix the problem. This message has been displayed for a long time.

**Comment by raphaelm** [comment, marked as answer]
you can ignore the message. pretix has been sca-compatible for years, I think since about 3.0. I have no idea why the Stripe message does not go away. I think they only know you use pretix but not what version.
